package.json

```diff
@@ -21,5 +21,6 @@
   "dependencies": {
     "@docsearch/js": "3",
     "@hotwired/turbo-rails": "^7.3.0"
-  }
+  },
+  "version": "0.0.0"
```

Comment:
What's the purpose of this `package.json`? If this was unintentional, could it please be removed?

Reply:
this change is unintentional and should not be part of the PR